doc/src/cylc-user-guide/cug.tex

```diff
@@ -4379,6 +4379,7 @@ \subsubsection{Parameter Expansion}
     [[parameters]]
         obs = ship, buoy, plane
         run = 1..5  # 1, 2, 3, 4, 5
+        idx = 1..9^2  # 1, 3, 5, 7, 9
```

Comment:
In fact, the `^` operator appears to be the bitwise XOR operator in most languages derived from C. The only other thing I can think of is to use `..` again in a ternary way, `START..END[..STEP]`, e.g. `1..9..2`.